experimenter/experimenter/experiments/api/v5/types.py

```diff
@@ -402,10 +402,11 @@ def resolve_conclusion_recommendations_choices(self, info):
     def resolve_population_sizing_data(self, info):
         try:
             sizing_data = cache.get(settings.SIZING_DATA_KEY)
-            if not isinstance(sizing_data, str):
-                raise TypeError(f"expected str, got {type(sizing_data).__name__}")
+            if sizing_data is not None:
```

Comment:
Omg I was literally thinking this all day yesterday I can't believe I forgot when reviewing XDDD